general-standards/README.md

```diff
@@ -1,24 +1,76 @@
 # AlliedCrowds General Standards
+
 > Let's play Russian Roulette:
 > `[ $[ $RANDOM % 6 ] == 0 ] && rm -rf / || echo *Click*`
 
+## Feature Lifecycle
+
+These are the stages that a new feature undergoes before being delivered. Collectively these can be considered "feature implementation":
+
+- **Initial conversation/Expression of need**: This is where the feature/issue is first expressed, it can be on a call, as a message, over email or in an issue itself
+- **Issue creation**: This is where it gets logged in the PM software
+- **Design**: For smaller features this is usually accomplished during coding. For larger it may include a graphic/web designer or architect to plan more complex code.
+- **Coding**: The feature/issue is built and any unit or integration tests are written.
+- **Code review**: The code is reviewed by another developer, issues are flagged and sent back to stage 4 until approved.
+- **Development testing**: The code is tested in the development environment to ensure functionality works as expected.
+- **Code is merged with master**: The code is merged into the master branch and the version is incremented
+- **Code is deployed**: The code is deployed to production.
+
+## Deployment Checklist
+
+This is the checklist that needs to be undertaken for a new feature before a new deployment is requested.
+
+### Code
+
+- [ ] Ensure [style guides](https://github.com/AlliedCrowds/styleguide) are followed, including git style guide
+- [ ] Format and lint code
+- [ ] Run tests locally
+- [ ] Ensure CI tests pass
+- [ ] Ensure commit messages are succinct
+- [ ] Check and resolve merge conflicts
+- [ ] Review PR in GitHub UI, this makes it much easier to check for miscellaneous print statements, debugging code, and any other silly mistakes
+- [ ] Remove any dead or unused code
+- [ ] Make sure you update `.env.example`, `requirements.txt`, etc. when the environment changes
+- [ ] Check all spelling and grammar within code
+- [ ] Ensure the variables have appropriate names, following convention
+- [ ] Does the PR reference the issue being resolved (e.g. `-#XYZ`)?
```

Comment:
My instinct is that instead of referencing the issue the PR closes this should be done in a comment at the end of the PR. This way you can list a number of issues you close and add details about how you've done so if necessary. Its pretty rare that i've made a PR that only closes one issue and having a `#n` in the branch name doesn't really inform me of the issue being resolved any better than the name does. The auto close issue on pr merge functionality works for comments like:

fixes #n, #m, #s 

So that's still a given and it means if we incorrectly name the branch or forget to add the issue number we won't have to mess around renaming it.